### ISSUE TITLE
subgroups in the sidebar look like groups with no avatar

### ISSUE THREAD

**Issue by juliencarnot** [loomio/loomio#4610]
![before](https://user-images.githubusercontent.com/3848493/38145535-12051188-344a-11e8-9e0a-86965bf3c119.png)

In the sidebar, subgroups are styled just like groups, but with no avatar shown. They do not appear to be linked or part of their parent, have no indentation, or font-size variation.

**Comment by juliencarnot** [comment]
after:
![after](https://user-images.githubusercontent.com/3848493/38145679-ddc2dfc6-344a-11e8-8cb1-64e8f6986ec5.png)

**Comment by gdpelican** [comment]
Hey @juliencarnot, I think the arrows add a bit too much iconography to the sidebar.

This has actually uncovered a missing piece in our static assets, which is that we don't actually serve the Roboto font (which the entire app uses) with the app, meaning we fall back to a system font if it's not installed on a client's machine. @robguthrie will fix this soon.

Here's a screenshot of what the sidebar looks like with Roboto installed:
![image](https://user-images.githubusercontent.com/750477/38218864-491584d4-3727-11e8-9714-f0cc37ec9b3e.png)

I'm hoping that once this fix goes in there will be less confusion around this. Does it seem like an improvement to you? Thanks again for your initiative, it definitely is a great help.

**Comment by juliencarnot** [comment]
Thanks for your feedback! I installed Roboto and see that the font-weight is different between groups and subgroups, but I still feel subgroups are visually floating around, which could be better with some indentation and font-size differentiation, if no icon is added.

I agree the arrow I proposed is a bit heavy, I tried a right chevron which is lighter:

![image](https://user-images.githubusercontent.com/3848493/38235260-c1b9c24e-3720-11e8-9b75-46c032185129.png)

I updated my branch, feel free to cherry-pick what you find interesting (this also fixes long subgroup names being cut off as in your capture).
